Update to v1.3

isuruf opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

isuruf committed:
test v1.3-RC1

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

inducer commented:
Thanks for working on this!

isuruf committed:
Pass --as-needed to the linker

isuruf committed:
Use -nodefaultlibs

Otherwise clang tries to add -lgcc and fails. This is probably an issue with our clang compiler and Anaconda's pseudo-cross compilers.

isuruf committed:
Update to RC3

isuruf committed:
Update to 1.3
